azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -215,6 +229,7 @@ func resourceArmCosmosDBAccountRead(d *schema.ResourceData, meta interface{}) er
 	d.Set("name", resp.Name)
 	d.Set("location", azureRMNormalizeLocation(*resp.Location))
 	d.Set("resource_group_name", resGroup)
+	d.Set("kind", string(resp.Kind))
```

Comment:
I would have thought `d.Set` can do the casting, but maybe not... 🤔

Reply:
we could, however I think it's worth being explicit that this value is coming from an external type?